analyze_internal_test.go

```diff
@@ -67,7 +67,7 @@ func Test_scriptAnalyzer(t *testing.T) {
 	fn := scriptAnalyzer(src)
 	deps, err := fn()
 
-	require.NoError(t, err)
+	require.Error(t, err)
```

Comment:
Isn't that right, that now we expect error? :sweat_smile:

Reply:
Yeah, I documented in the PR as a breaking change
> If the script source is given a name but not content, the name is used as the script's file and its content is tried to be read. Before this was not the case because it was assumed the content was already read always

Reply:
I see, yes makes totally sense!

Reply:
If I understand correctly, the reason `scripAnalyzer()` should return an error is because `script.js` does not exist. In this case, I would definitely not use such a file name, because it is possible that a file with such a name will exist in the future. In such cases, I either use a file named `I-am-sure-this-file-does-not-exist.js`, or create a temp directory and reference a non-existent file inside it.